docs/developers_guide/ocean/api.md

```diff
@@ -169,13 +103,40 @@
    analysis.Analysis.exact_solution
    analysis.Analysis.get_output_field
 
-   viz.VizMap
-   viz.VizMap.run
+   viz.Viz
+   viz.Viz.run
+```
+
+### inertial_gravity_wave 
+
+```{eval-rst}
+.. currentmodule:: polaris.ocean.tasks.inertial_gravity_wave
+
```

Comment:
I think we want to keep these headings alphabetized here in the API so they mimic the directory structure in the Polaris package.